### DISCUSSION TITLE
Design question

### DISCUSSION THREAD

**Discussion by eric859** [netbox-community/netbox#14994] [General]
Just got Netbox installed and the first thing I need to do is confusing me.  Not sure how to configure my Sites, Regions, Tenants, Location and all groups.

I work at a school district with 5 schools, each within the same town.  Each school as an MDF and several IDF's, and classrooms/gym/auditorium.  There are goups of employees by Admin, Teachers, Support Staff(Teacher aids, substitutes) and Facilities (Janitors, etc).

The heirarchy I'm thinking of is School District -> School -> IDF/MDF.  Then I guess I'll add racks, servers, maybe classrooms, etc...

How would I set this up using Sites, Regions and Tenants???

Thank you in advance!

**Comment by zmaster7** [comment]
[https://docs.netbox.dev/en/feature/getting-started/planning/#order-of-operations](https://docs.netbox.dev/en/feature/getting-started/planning/#order-of-operations)

The planning documentation above can give you an idea of the hierarchy, and drilling down into the various models from there will give you a description of it's intended use case. For example, drilling down into regions will bring you to [https://docs.netbox.dev/en/feature/models/dcim/region/](https://docs.netbox.dev/en/feature/models/dcim/region/) which describes a region as a geographical association to sites.

To give you a rough breakdown, your region would most likely be the town the schools are located in. In your use case, you said the schools are located in the same town, so filling out this information may or may not be beneficial to you.

Next up, site groups would be another optional grouping for you depending on future scope. This would be the most logical place to put the school district. If the scope of your NetBox instance is only ever going to cover one school district, it may not be beneficial.

After that, you would define your sites as each individual school, as this is the model that contains physical addresses of those districts. If you decided to create a site group and/or region, you would associate your districts with said site group/region.

The last piece of the physical associations before you start getting into racks/devices would be the location. Think of these as your rooms inside each school. Going further, you can have nested locations if you need finer detail for organization later on. For example, you could have a location named "Datacenter" and nested locations with "Row 1," "Row 2," etc.

As for tenants, there are a multitude of ways you can define these. You will have to decide what makes the most sense in your environment. For example, in my environment I use NetBox for internal billing between departments. Using internal budget codes was the logical choice for this. You can then use contacts to define who is responsible for which tenant, rack, device, etc.
